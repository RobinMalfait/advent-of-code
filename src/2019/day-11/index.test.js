import { promises } from 'node:fs'
import { resolve } from 'node:path'

import part1 from './part-1.js'
import part2 from './part-2.js'

let data = promises.readFile(resolve(__dirname, '..', '..', '..', 'data', '2019-11.txt'), 'utf8')

describe('Part 1', () => {
  // Actual test, Part 1
  it('should produce the correct value for the input data', async () => {
    expect(await part1(await data)).toMatchSnapshot()
  })
})

describe('Part 2', () => {
  // Actual test, Part 2
  it('should produce the correct value for the input data', async () => {
    expect(await part2(await data)).toMatchSnapshot()
  })
})
