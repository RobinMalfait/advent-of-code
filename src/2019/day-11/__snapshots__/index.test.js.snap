// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Part 1 should produce the correct value for the input data 1`] = `2392`;

exports[`Part 2 should produce the correct value for the input data 1`] = `
"
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
▒▒▒▒████████▒▒▒▒████▒▒▒▒██████▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒████████▒▒██▒▒▒▒██▒▒████████▒▒▒▒
▒▒▒▒████████▒▒▒▒████▒▒▒▒██████▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒████████▒▒██▒▒▒▒██▒▒████████▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒██████▒▒▒▒██▒▒▒▒▒▒▒▒██████▒▒▒▒████████▒▒██▒▒▒▒▒▒▒▒██████▒▒▒▒██▒▒▒▒██▒▒██████▒▒▒▒▒▒
▒▒▒▒██████▒▒▒▒██▒▒▒▒▒▒▒▒██████▒▒▒▒████████▒▒██▒▒▒▒▒▒▒▒██████▒▒▒▒██▒▒▒▒██▒▒██████▒▒▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒████▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒████▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒▒▒▒▒██▒▒▒▒██▒▒██▒▒▒▒▒▒▒▒▒▒
▒▒▒▒████████▒▒▒▒██████▒▒██████▒▒▒▒██▒▒▒▒██▒▒████████▒▒████████▒▒▒▒████▒▒▒▒████████▒▒▒▒
▒▒▒▒████████▒▒▒▒██████▒▒██████▒▒▒▒██▒▒▒▒██▒▒████████▒▒████████▒▒▒▒████▒▒▒▒████████▒▒▒▒
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
"
`;
